app/_how-tos/rate-limit-based-on-peak-time.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Rate limit requests based on peak and off-peak time
+content_type: how_to
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+tools:
+    - deck
+
+prereqs:
+  entities:
+    services:
+        - example-service
+
+min_version:
+  gateway: '3.4'
+
+plugins:
+  - rate-limiting-advanced
+  - pre-function
+
+entities:
+  - service
+  - route
+  - plugin
+
+tags:
+  - rate-limiting
+  - serverless
+
+tldr:
+  q: How do I set different rate limits to handle traffic during peak and off-peak time?
+  a: |
+    You can set the rate limit based on peak or non-peak time by using the Pre-function and the Rate Limiting Advanced plugins together.
+
+    This tutorial shows you how to handle traffic with two different Routes: one for peak traffic, and one for off-peak traffic. Then, you apply two plugins:
+    * The Rate Limiting Advanced plugin applies a different rate limit to each Route.
+    * The Pre-function plugin runs a Lua function in the rewrite phase, sending traffic to one of these Routes based on the defined peak and off-peak settings.
```

Comment:
```suggestion
    * The Pre-function plugin runs a Lua function in the rewrite phase, sending traffic to one of these Routes based on the defined peak and off-peak settings in the headers.
```
I wanted a bit more info here about how this is configured, and I think it was missing this bit about how this is controlled via the headers that are set.